docs/architecture/flags.md

```diff
@@ -1,6 +1,6 @@
 # Flags
 
-The `Flags` interface provides a simple API for storing and retrieving launcher flags at runtime. Currently, flags are of types `bool`, `int64` and `string`.
+The `Flags` interface provides a simple API for storing and retrieving launcher flags at runtime.
```

Comment:
We have implemented more flag types so I don't think it's helpful to enumerate them here anymore